pkg/controller/container-runtime-config/helpers.go

```diff
@@ -58,17 +61,21 @@ type tomlConfigCRIO struct {
 
 type updateConfigFunc func(data []byte, internal *mcfgv1.ContainerRuntimeConfiguration) ([]byte, error)
 
-func createNewCtrRuntimeConfigIgnition(storageTOMLConfig, crioTOMLConfig []byte) igntypes.Config {
+// createNewIgnition returns the data from the ignition files
+func createNewIgnition(configs map[string][]byte) igntypes.Config {
```

Comment:
(Absolutely non-blocking, for a separate PR: This could be moved into a shared helper, maybe somewhere next to `NewIgnConfig`, and called also from `pkg/controller/kubelet-config`.)